### DISCUSSION TITLE
Issue with Supabase Vector on Coolify (Raspberry Pi 5)

### DISCUSSION THREAD

**Discussion by FrancescoBrunoDev** [coollabsio/coolify#3734] [Q&A]
Hello everyone,  
I am trying to get Supabase running on Coolify on a Raspberry Pi 5. The issue is with Supabase Vector, particularly the `supabase-vector` container, which is returning an error that prevents everything from working.

```bash
2024-10-05T16:14:14.841615Z ERROR vector::topology::builder: Configuration error. error=Transform "rest_logs": 
error[E105]: call to undefined function
  ┌─ :4:18
  │
4 │     .timestamp = to_timestamp!(parsed.time)
  │                  ^^^^^^^^^^^^
  │                  │
  │                  undefined function
  │                  did you mean "is_timestamp"?
  │
  = learn more about error code 105 at https://errors.vrl.dev/105
  = see language documentation at https://vrl.dev
  = try your code in the VRL REPL, learn more at https://vrl.dev/examples
```

Does anyone have any ideas on how to solve this?

Thanks!

**Comment by rechbe200163** [comment]
Hello, same issue hear just tried to setup everthing up again nothing always the same issue 

log from the failing docker:
```
2024-12-06T18:49:17.876441423Z memory allocation of 5 bytes failed
2024-12-06T18:50:18.348513573Z <jemalloc>: Unsupported system page size
2024-12-06T18:50:18.348579388Z <jemalloc>: Unsupported system page size
2024-12-06T18:50:18.348584443Z memory allocation of 5 bytes failed
```

have you fixed yours by now?

**Reply by FrancescoBrunoDev** [reply]
Hi @rechbe200163 ,
Thank you for reminding me about this thread. Yes, I found a solution, and you can find it explained here:
[https://pimylifeup.com/raspberry-pi-page-size/]()

It's a straightforward procedure, and you can revert it without any issues.

It could be necessary to change the to_timestamp thing, though.

There was a pull request on the Supabase repo: [https://github.com/supabase/supabase/pull/28931]()

PS: However, I must tell you that when I tested it on a Raspberry Pi 5, the performance was really poor, likely due to Supabase's AI features.

**Reply by hismayilov** [reply]
do following change in vector.yml
in
 /data/coolify/services/XXXXXXXXXXXX/volumes/logs
-      .timestamp = to_timestamp!(parsed.time)
+      .timestamp = parse_timestamp!(parsed.time, format: "%+")

<img width="1639" height="290" alt="CleanShot 2026-04-28 at 11 50 25" src="https://github.com/user-attachments/assets/e499f025-d367-46c3-af51-ba665d2ae576" />
